diff --git a/node_modules/@simonsmith/cypress-image-snapshot/command.d.ts b/node_modules/@simonsmith/cypress-image-snapshot/command.d.ts
index 2dc3f87..1f975e7 100644
--- a/node_modules/@simonsmith/cypress-image-snapshot/command.d.ts
+++ b/node_modules/@simonsmith/cypress-image-snapshot/command.d.ts
@@ -1,7 +1,7 @@
-import type { CypressImageSnapshotOptions } from './types';
+// import type { CypressImageSnapshotOptions } from './types';
 /**
  * Add this function to your `supportFile` for e2e/component
  * Accepts options that are used for all instances of `toMatchSnapshot`
  */
-export declare const addMatchImageSnapshotCommand: (defaultOptionsOverrides?: CypressImageSnapshotOptions) => void;
+export declare const addMatchImageSnapshotCommand: (defaultOptionsOverrides?: any) => void;
 //# sourceMappingURL=command.d.ts.map
