diff --git a/node_modules/@vue/runtime-core/dist/runtime-core.d.ts b/node_modules/@vue/runtime-core/dist/runtime-core.d.ts
index d337174..8d7ae75 100644
--- a/node_modules/@vue/runtime-core/dist/runtime-core.d.ts
+++ b/node_modules/@vue/runtime-core/dist/runtime-core.d.ts
@@ -1104,7 +1104,7 @@ export type SetupContext<E = EmitsOptions, S extends SlotsType = {}> = E extends
     attrs: Data;
     slots: UnwrapSlotsType<S>;
     emit: EmitFn<E>;
-    expose: (exposed?: Record<string, any>) => void;
+    expose: <Exposed extends Record<string, any> = Record<string, any>>(exposed?: Exposed) => void
 } : never;
 /* removed internal: InternalRenderFunction */
 /**
